internal/persistence/persistors/mysqlstore/store_organization.go

```diff
@@ -0,0 +1,290 @@
+package mysqlstore
+
+import (
+	"context"
+	"fmt"
+	"github.com/dembygenesis/local.tools/internal/model"
+	"github.com/dembygenesis/local.tools/internal/persistence"
+	"github.com/dembygenesis/local.tools/internal/persistence/database_helpers/mysql/assets/mysqlmodel"
+	"github.com/dembygenesis/local.tools/internal/persistence/database_helpers/mysql/mysqltx"
+	"github.com/dembygenesis/local.tools/internal/sysconsts"
+	"github.com/friendsofgo/errors"
+	"github.com/volatiletech/sqlboiler/v4/boil"
+	"github.com/volatiletech/sqlboiler/v4/queries/qm"
+)
+
+func (m *Repository) DeleteOrganization(
+	ctx context.Context,
+	tx persistence.TransactionHandler,
+	id int,
+) error {
+	ctxExec, err := mysqltx.GetCtxExecutor(tx)
+	if err != nil {
+		return fmt.Errorf("get ctx exec: %v", err)
+	}
+
+	entry := &mysqlmodel.Organization{
+		ID:       id,
+		IsActive: false,
+	}
+	if _, err = entry.Update(ctx, ctxExec, boil.Whitelist("is_active")); err != nil {
+		return fmt.Errorf("delete: %v", err)
+	}
+
+	return nil
+}
+
+func (m *Repository) UpdateOrganization(ctx context.Context, tx persistence.TransactionHandler, params *model.UpdateOrganization) (*model.Organization, error) {
+	if params == nil {
+		return nil, ErrOrgNil
+	}
+	ctxExec, err := mysqltx.GetCtxExecutor(tx)
+	if err != nil {
+		return nil, fmt.Errorf("extract context executor: %v", err)
+	}
+
+	entry := &mysqlmodel.Organization{ID: params.Id}
+	cols := []string{mysqlmodel.OrganizationColumns.ID}
+
+	if params.Name.Valid {
+		entry.Name = params.Name.String
+		cols = append(cols, mysqlmodel.OrganizationColumns.Name)
+	}
+
+	_, err = entry.Update(ctx, ctxExec, boil.Whitelist(cols...))
+	if err != nil {
+		return nil, fmt.Errorf("update failed: %v", err)
+	}
+
+	organization, err := m.GetOrganizationById(ctx, tx, entry.ID)
+	if err != nil {
+		return nil, fmt.Errorf("get organitzation by id: %v", err)
+	}
+
+	return organization, nil
+}
+
+func (m *Repository) GetOrganizationById(ctx context.Context, tx persistence.TransactionHandler, id int) (*model.Organization, error) {
+	paginated, err := m.GetOrganizations(ctx, tx, &model.OrganizationFilters{
+		IdsIn: []int{id},
+	})
+	if err != nil {
+		return nil, fmt.Errorf("organization filtered by id: %v", err)
+	}
+
+	if paginated.Pagination.RowCount != 1 {
+		return nil, fmt.Errorf(sysconsts.ErrExpectedExactlyOneEntry, id)
+	}
+
+	return &paginated.Organizations[0], nil
+}
+
+func (m *Repository) GetOrganizationByName(ctx context.Context, tx persistence.TransactionHandler, name string) (*model.Organization, error) {
+	paginated, err := m.GetOrganizations(ctx, tx, &model.OrganizationFilters{
+		OrganizationNameIn: []string{name},
+	})
+	if err != nil {
+		return nil, fmt.Errorf("organization filtered by name: %v", err)
```

Comment:
use %w, instead of %v

corrected:
 fmt.Errorf("organization filtered by name: **%w**", err)
Comment

Reply:
does this apply to all fmt.Errorf ?